### PR TITLE
Webview2 SDK 1.0.774.44

### PR DESCRIPTION
**Pull request by WilliamVenner**

### CODE REVIEW

File: src/interfaces.rs [hunk 1]
```diff
@@ -24,6 +24,54 @@ impl WebView {
     }
 }
 
+/// Wrapper for `ICoreWebView2_2`.
+#[derive(Clone)]
+pub struct _2 {
```
Comment (WilliamVenner) [inline src/interfaces.rs:29]: This is super annoying. I'm not really sure what to do about these. It's not actually clear on how to use the suffixed interfaces.

File: webview2-sys/build.rs [hunk 2]
```diff
@@ -1,14 +1,18 @@
 use std::path::PathBuf;
 
+const WEBVIEW_SDK_VERSION: &str = "2.1.0.774.44";
```
Comment (sopium) [inline webview2-sys/build.rs:3]:
```suggestion
const WEBVIEW_SDK_VERSION: &str = "1.0.774.44";
```

File: webview2-sys/build.rs [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
+
     if std::env::var("CARGO_CFG_TARGET_OS").unwrap() != "windows" {
         return;
     }
     let is_msvc = std::env::var("CARGO_CFG_TARGET_ENV").map_or(false, |env| env == "msvc");
 
     let manifest_dir = std::env::var("CARGO_MANIFEST_DIR").unwrap();
     let mut manifest_dir = PathBuf::from(manifest_dir);
-    manifest_dir.push("Microsoft.Web.WebView2.1.0.622.22");
+    manifest_dir.push("Microsoft.Web.WebView".to_owned() + WEBVIEW_SDK_VERSION);
```
Comment (sopium) [inline webview2-sys/build.rs:15]:
```suggestion
    manifest_dir.push("Microsoft.Web.WebView2.".to_owned() + WEBVIEW_SDK_VERSION);
```

File: webview2-sys/idl2rs/src/idl.pest [hunk 4]
```diff
@@ -27,7 +27,8 @@ method_name = { identifier }
 method = { doc_comment? ~ method_attribute? ~ _type ~ method_name ~ "(" ~ parameters ~ ")" ~ ";" }
 
 // Enum.
-variant = { doc_comment? ~ identifier }
+variant_value = { ("0x" ~ (('A' .. 'F') | ('0' .. '9'))+) | digit+ }
+variant = { doc_comment? ~ identifier ~ (WHITESPACE? ~ "=" ~ WHITESPACE? ~ variant_value ~ WHITESPACE?)? }
```
Comment (sopium) [inline webview2-sys/idl2rs/src/idl.pest:31]: `WHITESPACE` shouldn't need to explicitly specified?

https://pest.rs/book/grammars/syntax.html#implicit-whitespace

File: src/interfaces.rs (hunk 1 above)
Comment (sopium) [reply src/interfaces.rs:29]: Should be similar to this:

https://github.com/sopium/webview2/commit/ca5dbd7959b030ed574b548457c6c16246284a7d#diff-2dc46f450fd671c92abaecb89b2b6842af16747d8a5de2082ae4663c07697b88R136-R153

Once you have a `WebView`, you should be able to use `as_inner` and `get_interface` to obtain an instance of `_2`(`ICoreWebview2_2`).

**Review by sopium — CHANGES_REQUESTED** [review]

File: src/interfaces.rs (hunk 1 above)
Comment (sopium) [reply src/interfaces.rs:29]: I shall add some more friendly APIs for these later. (And give them some better names...)

**Comment by sopium** [comment]
Modified, squashed and merged from the command line.

**Pull request closed without merging**